power_grid_model_c/power_grid_model/include/power_grid_model/optimizer/tap_position_optimizer.hpp

```diff
@@ -354,6 +354,9 @@ template <transformer_c... TransformerTypes> class TransformerWrapper {
     IntS tap_max() const {
         return apply([](auto const& t) { return t.tap_max(); });
     }
+    IntS tap_range() const {
+        return apply([](auto const& t) { return std::abs(t.tap_max() - t.tap_min()); });
```

Comment:
this will wrap around if `|tap_min - tap_max| >= 128`... not sure what would be a good solution though

Reply:
could use some upgrade ;)